polars/polars-arrow/src/bitmap.rs

```diff
@@ -0,0 +1,82 @@
+use crate::bit_util::get_bit_raw;
+use crate::trusted_len::TrustedLen;
+use arrow::bitmap::Bitmap;
+
+pub trait AddIterBitmap {
+    /// constructs a new iterator
+    fn iter(&self) -> BitmapIter<'_>;
+}
+
+impl AddIterBitmap for Bitmap {
+    fn iter(&self) -> BitmapIter<'_> {
+        let slice = self.buffer_ref().as_slice();
+        BitmapIter::new(slice, 0, self.len())
+    }
+}
+
+/// An iterator over bits according to the [LSB](https://en.wikipedia.org/wiki/Bit_numbering#Least_significant_bit),
+/// i.e. the bytes `[4u8, 128u8]` correspond to `[false, false, true, false, ..., true]`.
+#[derive(Debug, Clone)]
+pub struct BitmapIter<'a> {
```

Comment:
was it not possible to use arrow2's here (or is this the master branch?)

Reply:
This is the master branch and its just an exact copy paste of your code.😉 I forgot the forked comment. Will add that.